terraform/install.sh

```diff
@@ -174,6 +174,32 @@ applyTerraform() {
   terraform apply -auto-approve -var="billing_account=${billing_acct}" -var="project_id=${project_id}" -var="bucket_name=${bucket_name}"
 }
 
+installMonitoring() {
+  log "Retrieving the external IP address of the application..."
+  TRIES=0
+  external_ip="";
+  while [[ -z $external_ip && "${TRIES}" -lt 20 ]]; do
+     external_ip=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.status.loadBalancer.ingress[0].ip}'); 
+     [ -z "$external_ip" ] && sleep 5; 
+     TRIES=$((TRIES + 1))
+  done;
+
+  if [[ TRIES -eq 20 ]]; then
```

Comment:
To be safe I'd change this to `z $external_ip`, just in case it found it in the last iteration (or code changes).

It's good to check against the variable you care about, not derivative variable

Reply:
Gotcha. Changed :)